src/scripts/upload.sh

```diff
@@ -18,7 +18,7 @@ echo "coveralls installed"
 if ! command -v python &> /dev/null; then
   $SUDO apt update
   $SUDO apt -y upgrade
-  $SUDO apt install -y python
+  $SUDO apt install python-is-python3
```

Comment:
Is this tested?

Reply:
https://github.com/ergeon/erg-core-components/pull/621